source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -74,25 +74,23 @@ level: 4
 ref: install
 content: |
 
-  To install the latest stable version of ``mongosh``, issue the
-  following command:
+  ``mongosh`` supports OpenSSL. You can install ``mongosh`` without
```

Comment:
[suggestion] Could potentially put this blurb in an include given it's also used in the Debian instructions?

Reply:
Done